templates/tapestry.js

```diff
@@ -2581,17 +2583,17 @@ function tapestryTool(config){
         if (config.wpCanEditTapestry) {
             return true;
         }
-    
-        // CHECK 3: If node is the root node or is currently selected
-        if (node.nodeType === "root" || (node.id == tapestry.dataset.rootId && node.nodeType !== "")) return true;
 
-        // CHECK 4: If node is an accordion row and user is not the author
+        // CHECK 3: If node is an accordion row and user is not the author
         if (node.presentationStyle === "accordion-row" && !config.wpCanEditTapestry) return false;
 
-        // CHECK 5: Hide stage and question set nodes unless user is an editor - this check is for TYDE only
+        // CHECK 4: Hide stage and question set nodes unless user is an editor - this check is for TYDE only
         if ((node.tydeType === "Stage" || node.tydeType === "Question set") && !config.wpCanEditTapestry) {
             return false;
         }
+
+        // CHECK 5: If node is the root node or is currently selected
+        if (node.nodeType === "root" || (node.id == tapestry.dataset.rootId && node.nodeType !== "")) return true;
```

Comment:
What is the point of having this here when the next statement after this returns true anyways? The original point of this was to return `true` before other things returned `false`.

Reply:
This was causing a bug where when a copilot clicks on a topic to view in the module summary, that topic becomes viewable afterwards because it becomes the root. I see your point though, I'm not sure if just removing this would cause any issues but I don't remember experiencing any.